schema/common.json

```diff
@@ -45,6 +45,9 @@
         "timestamp",
         "level"
       ]
+    },
+    "families": {
+      "enum": [ "bacnet", "ipv4", "ipv6" ]
```

Comment:
Why isn't this   +=  [ "iot", "ethmac" ]   ?

Reply:
dead code -- wasn't used at all -- removed!